anta/cli/exec/commands.py

```diff
@@ -50,14 +50,15 @@ def _get_snapshot_dir(ctx: click.Context, param: click.Parameter, value: str) ->
     "-o",
     show_envvar=True,
     type=click.Path(file_okay=False, dir_okay=True, exists=False, writable=True),
-    help="Directory to save commands output. Will have a suffix with the format _YEAR-MONTH-DAY_HOUR-MINUTES-SECONDS'",
-    default="anta_snapshot",
+    help="Directory to save commands output.",
+    default=f"anta_snapshot_{datetime.now().strftime('%Y-%m-%d_%H_%M_%S')}",
     show_default=True,
     callback=_get_snapshot_dir,
```

Comment:
```suggestion
```

Reply:
Actually we need to preserve this function actual CI for the CLI